### PR TITLE
More fullnode -> validator renaming

### PR DESCRIPTION
**Pull request by mvines**
* fullnode.rs => validator.rs
* struct Fullnode => struct Validator

Part of #4409

### CODE REVIEW

File: book/src/SUMMARY.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
   - [Staking Delegation and Rewards](stake-delegation-and-rewards.md)
   - [Performance Metrics](performance-metrics.md)
 
-- [Anatomy of a Fullnode](fullnode.md)
+- [Anatomy of a Validator](fullnode.md)
```
Comment (garious) [inline book/src/SUMMARY.md]: Change the filename too?

Comment (garious) [reply book/src/SUMMARY.md]: hmm, after renaming validator.md to validator-proposed.md perhaps?

Comment (mvines) [reply book/src/SUMMARY.md]: ✔️

File: book/src/validator.md [hunk 2]
```diff
@@ -1,56 +1,29 @@
...
-* TPU moves to new socket-free crate called solana-tpu.
-* TPU's BankingStage absorbs ReplayStage
-* TVU goes away
-* New RepairStage absorbs Blob Fetch Stage and repair requests
-* JSON RPC Service is optional - used for debugging. It should instead be part
-  of a separate `solana-blockstreamer` executable.
-* New MulticastStage absorbs retransmit part of RetransmitStage
-* MulticastStage downstream of Blockstore
-
+<img alt="Validator block diagrams" src="img/fullnode.svg" class="center"/>
```
Comment (garious) [inline book/src/validator.md]: sneaky

**Pull request merged**